### ISSUE TITLE
`navset_card_*()` variants silently drop `header`/`footer` arguments

### ISSUE THREAD

**Issue by cpsievert** [rstudio/bslib#861]
`Header`/`Footer` should appear above/below the relevant `nav_panel()` content (just like they would in the `navset_tab()` case.

```r
navset_card_tab(
  header = "Header", 
  footer = "Footer", 
  nav_panel("A", "a"), 
  nav_panel("B", "b")
)
```

**Comment by vituri** [comment]
I confirm the problem with the footer argument.

**Comment by gadenbuie** [comment]
Also reported on Community: https://community.rstudio.com/t/trying-to-use-footer-argument-in-bslib-navset-card-tab/175577

**Comment by ideusoes** [comment]
Hello, any chance this will be fixed in the future ?

Given the nature of the dots arguments of `navset_card_*()` functions only accepting `nav_panel()`s, it makes it tricky and not user-friendly to add a footer.

**Comment by gadenbuie** [comment]
This was fixed by #1028, which is not yet released to CRAN but will be soon.
